packages/core/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@gitbeaker/core",
+  "description": "Full NodeJS implementation of the GitLab API. Supports Promises, Async/Await.",
+  "version": "14.0.1",
+  "author": {
+    "name": "Justin Dalrymple"
+  },
+  "browser": "dist/index.umd.js",
+  "bugs": {
+    "url": "https://github.com/jdalrymple/gitbeaker/issues"
+  },
+  "dependencies": {
+    "form-data": "^3.0.0",
+    "got": "^10.2.2",
+    "ky": "^0.16.1",
```

Comment:
something smells if we need to depend on both 🤔

Reply:
Yea, i couldnt think of the best way to split out the browser version. Its just a diff rollup config where i switch the deps from got to ky. If you have any suggestions id be open to trying something else?

Reply:
The only thing i could think of was having another package for the browser that used the core source with a custom Requester for Ky and its own rollup config. BUT that would require depending on the source of another project, which im not currently doing. Not sure if thats a good practice. Ill look up some examples is that done

Reply:
What about `@gitbeaker/core` with no dep to ky or got?
Than `@gitbeaker/node` with got and `@gitbeaker/browser` with ky?

Reply:
Then it even opens up to someone depends on `@gitbeaker/core` where they write their own requester.

Reply:
> What about `@gitbeaker/core` with no dep to ky or got?
> Than `@gitbeaker/node` with got and `@gitbeaker/browser` with ky?

This...this may be fine. Ill try it!

Reply:
This was a good idea! Thank you. Not sure if I should split out the infrastructure code into a helpers package because right now, the export for core is { all the services, all the bundles, and one helper class}. This requires deconstructuring if you want to import everything but not the helper. Not sure if its worth the hassle though.

It would also be nice to fix the path for the APIMap import which is now done though:

'import * as map from '@gitbeaker/core/dist/map.json'

Any ideas?

Reply:
Why not move the RequestHelper out to it's own package? x number of helpers?

You could also remove the RequestHelper from the `index.ts` and require that they import it directly.

Reply:
I split it off, its a little verbose, but more flexible

Reply:
As for the map thing, one suggestion is modifying the prepublish script to move dist files to root , and updating the files property of the package.json to include the few files i want exported. This would result in the export being:

```
'import * as map from '@gitbeaker/core/map.json'
```

Ill leave this for now, but something to think about